lib/resty/rsa.lua

```diff
@@ -388,7 +389,13 @@ function _M.new(_, opts)
         end
     end
 
-    local size = C.EVP_PKEY_size(pkey)
+    --local size = C.EVP_PKEY_size(pkey)
+    local size
+    if not pcall(function () return C.EVP_PKEY_size(pkey) end) then
```

Comment:
Let's use pcall like https://github.com/spacewander/lua-resty-rsa/blob/d4e22cfaa851eabbcc50029e20156ae236d733d8/lib/resty/rsa.lua#L151, so we only need to call once when loading the module.